src/app/projects/[ownerType]/[org]/[project]/cycles/[cycle]/page.js

```diff
@@ -283,8 +276,41 @@ export async function prepareData(params) {
     }
 
     const kind = item.fieldValues.nodes.find(fv => fv.field?.name === 'Kind')?.name
+    let scopes = scopeData[item?.content?.id] ? scopeData[item?.content?.id] : []
     const history = getHistory(item.content)
 
+    if (kind === 'Bet') {
+      if (scopes.length > 0) {
+        for (let [scopeIndex, scope] of scopes.entries()) {
+          scope.history = getHistory(scope)
+          scope.bet = item.content.url
+          scope.progress = {
+            percentage: scope.closed === true ? 100 : getCurrentPercentage(scope.comments.nodes.map(node => node.bodyText)),
+            history: scope.history,
+            notPlanned: getLatestCloseState(history) === 'NOT_PLANNED',
+            completed: getLatestCloseState(history) === 'COMPLETED',
+            closed: scope.closed === true
+          }
+          scope.color = colors[scopeIndex % colors.length]
+        }
+      } else {
+        // If scopes are not available, try with tasks
+        const extractedScope = extractTasks(item.content.body)
+        for (let [scopeIndex, scope] of extractedScope.entries()) {
+          scope.history = null;
+          scope.bet = item.content.url
+          scope.progress = {
+            percentage: scope.closed === true ? 100 : 0,
+            history: null,
+            notPlanned: false,
+            completed: scope.closed === true,
+            closed: scope.closed === true
+          },
+          scope.color = colors[scopeIndex % colors.length]
```

Comment:
Something here is not working. On `extractTasks` you're returning an array of objects with `title` and `progress` (`100%` or `0%`, including the `%` therefore a string, not a number). Also `scope.closed` doesn't make sense here since we don't have such information. We should be checking for the `progress` that you return on `extractTasks` but probably as a number, i.e., without the `%` at the end. I'm not sure but I'm suspicious this is the reason 😄

Reply:
@fmvilas 
If we are talking about this case bellow, then we consider that when the task is marked as done (`[x]`) it's a 100% - (I agree that it shouldn't be a string but a number, In that case we also asume that the scope is closed.
I'm not sure about how to get the progress that is nothing but 0% or 100%

<img width="903" alt="Screenshot 2023-06-21 at 10 08 35" src="https://github.com/shapeitapp/shapeitapp/assets/554438/37281e9b-5e11-4832-800b-98a4cfe54cba">

Reply:
@fmvilas I got your point, I've fixed in this commit fe1f42d597eb2d16efae4e4edfdd5e5c75865f40